dask/array/tests/test_fft.py

```diff
@@ -111,6 +111,7 @@ def test_wrap_bad_kind():
         fft_wrap(np.ones)
 
 
+@pytest.mark.slow
 @pytest.mark.parametrize("funcname", all_nd_funcnames)
 @pytest.mark.parametrize("dtype", ["float32", "float64"])
 def test_nd_ffts_axes(funcname, dtype):
```

Comment:
Ok, personally I'd still not like to see this test marked as slow. Tried cutting down the dimensionality of the test data for this test and it shaved ~3s off the test time of the FFT test suite. Please see PR ( https://github.com/dask/dask/pull/2270 ), which does exactly this.